### PR TITLE
[CI] fix timeout errors: increase timeout from 60m (default) to 90m

### PR DESCRIPTION
**Pull request by timotheecour**
I often run into CI timeouts, which causes more wasted time (both on PR author and on CI resources since whole CI needs to be re-run, even though most pipelines in the matrix finished except maybe 1 or 2 that timed out)

* this should fix most CI timeout errors, I didn't want to use a too high value (or 0 = max) in case some bad PR would run for a very long time, affecting other PR's; note that 80m wasn't enough (I still got timeout after 80m = 1h20 on this very PR)
* the diff is mostly re-indentation as you can see by running `git diff` locally
* this was the simplest change that actually works; modeled after https://github.com/kata-containers/packaging/commit/b58c6a4682346f821a9f0c23e5c4fe15ab492d4e ; see also https://github.com/microsoft/azure-pipelines-yaml/issues/48#issuecomment-525093222 which (as i verified) shows you can't simply put timeout under pool
* you can verify this timeout is taken into effect by replacing it by a small value (eg 2) and seeing the jobs cancelled after 2 minutes, which I did

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -1,148 +1,153 @@
...
-    displayName: 'Build koch'
-
-    # set result to omit the "bash exited with error code '1'" message
-  - bash: |
-      ./koch runCI || echo '##vso[task.complete result=Failed]'
-    displayName: 'Run CI'
-    env:
-      SYSTEM_ACCESSTOKEN: $(System.AccessToken)
+jobs:
+- job: packages
```
Comment (alaviss) [inline azure-pipelines.yml]: If there's only one job, this part is unneeded.

**Comment by alaviss** [comment]
AFAICT our max runtime with Azure has never actually touched that mark :/ This is kinda worrying and we should investigate the actual issue.

EDIT: It appears that it actually reached the mark for certain PRs, I guess I'll have to investigate this further.

**Comment by timotheecour** [comment]
ping: this keeps happening in lots of my PR's

**Comment by Araq** [comment]
Would have merged it days ago but you decided to rewrite everything and so looking at the diff is ineffective.

**Comment by timotheecour** [comment]
> you decided to rewrite everything

that's the only way that worked that I could find due to limitations I mentioned in top post

> looking at the diff is ineffective

that's why I wrote this: https://github.com/nim-lang/Nim/pull/10435 : see https://nim-lang.github.io/Nim/contributing.html#the-git-stuff-code-reviews
```
git fetch origin pull/13134/head && git checkout FETCH_HEAD
git diff --color-moved-ws=allow-indentation-change --color-moved=blocks HEAD^
```

## nit: should be `git diff`, not `git show`, i need to fixup https://github.com/nim-lang/Nim/pull/10435

it shows a tiny part changed, and everything else moved as a bock using color coding:

image

but best to see for yourself; it's the only reliable way i know of to look at large diffs where there is mostly moves / reindents

**Pull request merged**